models/_generate_layers.py

```diff
@@ -81,7 +81,8 @@ def setup():
         if param.is_module_init_arg():
           print(param.get_module_param_docstring(indent="    "), file=f)
       print('    """', file=f)
-      print(f"    super().__init__({'**kwargs' if issubclass(cls_base, LayerBase) else ''})", file=f)
+      params = get_super_call_kwargs_repr(layer_class)
+      print(f"    super().__init__({params})", file=f)
```

Comment:
I think I would refactor the function such that you do directly:
```suggestion
      print(f"    {sig.get_super_call_code_str()}", file=f)
```
This is cleaner, and also makes the documentation of the function `get_super_call_code_str` simpler and cleaner and more clear.